package.json

```diff
@@ -26,7 +26,7 @@
     "start": "node ./node_modules/react-native/local-cli/cli.js start --config ./rn-cli.config.js",
     "run-playground-ios": "cd examples/NavigationPlayground && react-native run-ios",
     "run-playground-android": "cd examples/NavigationPlayground && react-native run-android",
-    "test": "BABEL_ENV=development jest",
```

Comment:
This was not necessary. Actually, with this line, the `test` env in `.babelrc` was totally ignored.